colin/checks/labels/build-date.py

```diff
@@ -0,0 +1,15 @@
+from colin.checks.abstract.labels import LabelCheck
+
+
+class BuildDateLabelCheck(LabelCheck):
```

Comment:
This is set automatically, based on the our document. I don't know if it make sense to have it in Fedora.